[WIP] Update README.md

Ballyglunin1 opened the pull request:
Updated the README.md file with instructions on how to play the game for the players. This addresses issue #3.

Can you @rowenaryan review the instructions and ensure that they make sense.

Ballyglunin1 committed:
Update README.md

Updated the README.md file with instructions on how to play the game.

rowenaryan commented on README.md:
```diff
@@ -2,12 +2,14 @@
+3. Use the left & right arrow keys to position the shape. 
+4. The goal is to complete rows with no empty spaces.
+5. Whwn completed, the rows will disappear. 
 
```
We should tell the player how to pause the game. 

```suggestion
6. To pause the game, just press the space bar again.
```

rowenaryan commented on README.md:
```diff
@@ -2,12 +2,14 @@
+1. Press the space bar to begin.
+2. Uee the up & down arrow keys to rotate the shape.
+3. Use the left & right arrow keys to position the shape. 
+4. The goal is to complete rows with no empty spaces.
```
I think it would be a good idea to move the goal of the game to the start of the instructions.

rowenaryan reviewed (changes requested):
Hi @Ballyglunin1, I have finished reviewing the instructions and recommended some changes for you to consider.

Ballyglunin1 committed:
Update README.md

Co-authored-by: rowenaryan <79329072+rowenaryan@users.noreply.github.com>

Ballyglunin1 committed:
Update README.md

Made the suggested changes to the instructions on how to play the game and clearly defined the goal of the game.

rowenaryan reviewed (commented):
The instructions look complete now. Thanks for the great work @Ballyglunin1 .
